examples/nats-streaming-queue-group/subscriber.out

```diff
@@ -0,0 +1,10 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file, and execute the `ballerina run` command.
+# `queue-group.bal` contains three services belonging to the same
+# queue group.
+# When several messages are published, it can be noticed that
+# each message is received by only one queue group member.
```

Comment:
```suggestion
# each message is received by only one group member of the queue.
```

Reply:
`queue group` is a well known term for NATS. I don't think we should rephrase this way.